src/main/java/org/dependencytrack/persistence/FindingsQueryManager.java

```diff
@@ -194,15 +194,9 @@ public Analysis makeAnalysis(Component component, Vulnerability vulnerability, A
         // to null will wipe them. That is not the expected behavior when an AnalysisRequest
         // has some fields unset (so they're null). If fields are not set, there shouldn't
         // be any modifications to the existing data.
-        if (analysisState != null) {
-            analysis.setAnalysisState(analysisState);
-        }
-        if (analysisJustification != null) {
-            analysis.setAnalysisJustification(analysisJustification);
-        }
-        if (analysisResponse != null) {
-            analysis.setAnalysisResponse(analysisResponse);
-        }
+        analysis.setAnalysisState(analysisState);
+        analysis.setAnalysisJustification(analysisJustification);
+        analysis.setAnalysisResponse(analysisResponse);
```

Comment:
Any reason for making this change? As per comment above, this could unintentionally nuke existing values.

Reply:
This was a change from static analysis from IntelliJ, and I applied it since it was related. It's clearly an unnecessary change and I will be happy to revert it.